crates/playground/.gitignore

```diff
@@ -2,4 +2,5 @@
 **/*.rs.bk
 Cargo.lock
 bin/
+coverage/
```

Comment:
Seems inconsistent with the other gitignore where all the pathes are prefixed with `/`.